include/amici/steadystateproblem.h

```diff
@@ -204,6 +229,16 @@ class SteadystateProblem {
     bool initializeBackwardProblem(Solver *solver, Model *model,
                                    const BackwardProblem *bwd);
 
+    /**
+     * initialize backward computation by setting state, time, adjoint
+     * state and checking for preequilibration mode
+     *
+     * @param model pointer to the model object
+     * @param yQ vector to be multiplied with dxdotdp
+     * @param yQB resulting vector after multiplication
+     */
+    void getQBfromQ(Model *model, const AmiVector &yQ, AmiVector &yQB);
```

Comment:
Is there some better name for this? Personally don't like it...

Reply:
I think its fine as-is. maybe replace `get` with `compute` since its technically not a getter method.